website/content/en/blog/2024-03-15-green-reviews-working-group-moving-towards-measuring-the-sustainability-footprint-of-cncf-projects.md

```diff
@@ -0,0 +1,113 @@
+
+The destination where to publish the sustainability metrics is not decided upon at the point of writing this blog post. [CNCF devstats](https://devstats.cncf.io/) is being discussed as another possible place where to make this type of data publicly available.
+
+### Next steps for the sustainability footprint assessments and benchmark test pipeline
+
+The continuous operation of a Kubernetes cluster raises questions about how sustainable this solution is. Some decisions were forced, like using bare metal servers to be able to retrieve accurate energy metrics. But new questions keep coming up as the work on the technical platform progresses.
+
+Can we create nodes on-demand when conducting sustainability footprint assessments instead of running the nodes all the time? Another enhancement opportunity is that currently, Falco is always deployed and the kernel is always being tested, which we hope to improve after KubeCon + CloudNativeCon Europe 2024, so that we use the minimal amount of energy in the pipeline itself. These questions are crucial to the WG’s mission, and we are looking forward to the community's suggestions and contributions. 
+
+In addition, the sustainability footprint assessment itself will be something that we need to tackle in the next milestone. The WG’s goal is to make the sustainability footprint assessment pipeline useful for CNCF project maintainers, such as Falco maintainers. We will continue to collaborate with the respective project maintainers as we enter the “review” phase of the project. This type of sustainability footprint assessment is inspired by the processes like TAG Security’s Security Assessment (TSSA) that is already utilized for CNCF projects. More information about the collaboration between Green Reviews WG and CNCF project maintainers can be found in the [docs](https://github.com/cncf-tags/green-reviews-tooling/tree/main/docs/cncf-projects). **CNCF project maintainers who are interested in having their projects assessed are welcome to flag their interest [here](https://github.com/cncf/tag-env-sustainability/issues/223)!**
+
+Please keep in mind that only some components of the described vision are in production at the time of writing this blog post. If you would like to find out more about the latest status of the work that’s being done in the Green Reviews WG, please join our community in the [Green Reviews WG Slack channel](https://cloud-native.slack.com/archives/C060EDHN431) or the [TAG ENV Slack channel](https://cloud-native.slack.com/archives/C03F270PDU6), or check out the [documentation](https://github.com/cncf-tags/green-reviews-tooling/tree/main/docs) on GitHub.
+
+## Reflecting on Green Reviews WG's journey till now
+
+Since its formal announcement at KubeCon + CloudNativeCon North America 2023, the **Green Reviews Working Group** (WG) has been on a journey, making progress in the realm of sustainability within the cloud native ecosystem.
```

Comment:
```suggestion
Since its formal announcement at KubeCon + CloudNativeCon North America 2023, the Green Reviews Working Group (WG) has been on a journey, making progress in the realm of sustainability within the cloud native ecosystem.
```